vendor/k8s.io/kubernetes/staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -510,6 +510,17 @@ func (r *Request) SetHeader(key, value string) *Request {
 	return r
 }
 
+func (r *Request) SetHeaderValues(key string, values []string) *Request {
```

Comment:
Yeah good idea, I'll try to see if I can easily do that, sounds like a better way.